README.md

```diff
@@ -26,6 +26,8 @@ With lakeFS you can build repeatable, atomic and versioned data lake operations
 
 lakeFS supports AWS S3 or Google Cloud Storage as its underlying storage service. It is API compatible with S3, and works seamlessly with all modern data frameworks such as Spark, Hive, AWS Athena, Presto, etc.
 
+lakeFS is an truely free open source platform where you can share your knowledge and many more.
+
```

Comment:
I am unsure that this added messaging currently adds value to our users beyond what we already have in [contributing.md](https://github.com/treeverse/lakeFS/blob/master/docs/contributing.md).  As such I am reluctant to lengthen our README to include this.  If you do believe it is important, please split it into a separate PR so that you can add in its description how this improves.